README.md

```diff
@@ -143,33 +150,27 @@ Previous spec:
 }
 ```
 
-New spec:
-
-```json
+After:
+```jsonc
 {
     "name": "uploader",
-    "requestURL": "https://uploader.com/upload",
```

Comment:
all examples have the url part request url uppercase so it would be nice to keep it that way or lowercase every property for consistency